docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md

```diff
@@ -2062,4 +2066,5 @@ This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses
 - SPDX-FileCopyrightText: 2023, 2024 BASF SE
 - SPDX-FileCopyrightText: 2023, 2024 Henkel AG & Co. KGaA
 - SPDX-FileCopyrightText: 2023, 2024 Contributors to the Eclipse Foundation
+- SPDX-FileCopyrightText: 2023, 2024 MHP Management- und IT-Beratung GmbH (On behalf of Volkswagen AG)
```

Comment:
how comes VW / MHP is popping-up here?

Reply:
Did they participated in the review?

Reply:
Actually, I am from MHP, and I participate in the expert group (and also wrote the whitpaper) on behalf of Volkswagen AG.

Reply:
Ah, I am sorry, I am only used to the consortia team

Reply:
Yep but this on behalf is not correct

Reply:
Add or both companies, or none

Reply:
@FabianKentsch if you are working for Volkswagen AG then volkswagen needs to be included.

And they should clarify if MHP shall be included. Example: CGI was working for BMW, and BMW Allowed CGI to be listed.